### PR TITLE
chore: prepare codemod for error prop

### PR DESCRIPTION
**Pull request by ozgurkececioglu**
[FX-2502]

### Description

- prepared a codemod for the input field components which uses `error` as a prop

### Review

- [ ] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [ ] Annotate all `props` in component with documentation
- [ ] Create `examples` for component
- [ ] Ensure that deployed demo has expected results and good examples
- [ ] Ensure that tests pass by running `yarn test`
- [ ] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [ ] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-2502]: https://toptal-core.atlassian.net/browse/FX-2502?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by denieler — APPROVED** [review]

**Review by denieler — COMMENTED** [review]
thanks for removing `packages/picasso-codemod/types.ts` 🙇 👍

**Comment by ozgurkececioglu** [comment]
@toptal-bot run package:alpha-release

**Review by OndrejTuma — APPROVED** [review]
Awesome man 💪 
Please just add info to README as well 🙏

**Review by TomasSlama — APPROVED** [review]

File: .prettierignore [hunk 1]
```diff
@@ -25,3 +25,5 @@ packages/picasso-codemod/src/v17.0.0/container-borders/__testfixtures__/expected
 packages/picasso-codemod/src/v17.0.0/container-borders/__testfixtures__/actual.tsx
 packages/picasso-codemod/src/v18.0.0/picasso-lab/__testfixtures__/basic.input.tsx
 packages/picasso-codemod/src/v18.0.0/picasso-lab/__testfixtures__/basic.output.tsx
+packages/picasso-codemod/src/v20.0.0/replace-error/__testfixtures__/as-variable.input.tsx
+packages/picasso-codemod/src/v20.0.0/replace-error/__testfixtures__/as-variable.output.tsx
```
Comment (ozgurkececioglu) [inline .prettierignore:29]: After running codemod in this example file, it produces `;` at the end of return block. To not fail the tests, I had to add them to ignore

**Comment by ozgurkececioglu** [comment]
@toptal-bot run package:alpha-release

**Comment by ozgurkececioglu** [comment]
@toptal-bot run all

File: packages/picasso-codemod/src/v20.0.0/replace-error/__testfixtures__/as-variable.output.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        autoFocus
+        status={error ? 'error' : 'default'}
+        placeholder={placeholder}
+        required={required}
+        size='small'
+        value={value}
+        onChange={onChange}
+      />
+      <NumberInput
+        status={hasError || valueValidationFailed ? 'error' : 'default'}
```
Comment (OndrejTuma) [inline packages/picasso-codemod/src/v20.0.0/replace-error/__testfixtures__/as-variable.output.tsx:24]: 👍

**Review by OndrejTuma — APPROVED** [review]
LGTM 👌

**Review by TomasSlama — APPROVED** [review]

**Review by denieler — APPROVED** [review]

**Pull request merged**